Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -84,9 +90,29 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $em->getConnection()->connect($input->getOption('shard'));
         }
 
-        $fixtures = $this->fixturesLoader->getFixtures();
+        $dirOrFile = $input->getOption('fixtures');
+        if ($dirOrFile) {
+            $paths = is_array($dirOrFile) ? $dirOrFile : array($dirOrFile);
+        } else {
+            /** @var $kernel \Symfony\Component\HttpKernel\KernelInterface */
+            $kernel = $this->getApplication()->getKernel();
+            $paths = array($kernel->getRootDir().'/DataFixtures');
+            foreach ($kernel->getBundles() as $bundle) {
+                $paths[] = $bundle->getPath().'/DataFixtures';
+            }
+        }
+
+        $loader = new DataFixturesLoader($this->getContainer());
+        foreach ($paths as $path) {
+            if (is_dir($path)) {
+                $loader->loadFromDirectory($path);
+            } elseif (is_file($path)) {
+                $loader->loadFromFile($path);
+            }
+        }
+        $fixtures = $loader->getFixtures();
```

Comment:
with this, you can't load services fixtures, and are forcing the user to store his fixtures in `<root>/DataFixtures` or even using bundles